### PR TITLE
Updated complex_words.csv's location and updated complexity_df function

### PR DESCRIPTION
**Pull request by BritVandy**
# Description

-  Changed the call location from my temp repo to the Scribble repo for the complex_words.csv
- Updated the complexity_df function to include dividing by how many times the word is used in the story
- Fixed the complex_words.csv, had a few missing items

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [X] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

[Loom Video](https://www.loom.com/share/90515838d7f34a2293563d58eb8593b1)
[Rubric](https://www.notion.so/lambdaschool/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)
[Trello Card](https://trello.com/c/wfHY2Hsf)

### CODE REVIEW

**Comment by singparvi** [comment]
Hi Brittany nice work. Just a small suggestion. The link you have for the complex_words CSV could be done local to the repo something like '../data/crop_cloud'. Great work updating the function. Nice work updating the complex_words.csv for scores. I think the pull request is reasonable enough and easy to go through and not too overwhelming to comprehend.

**Review by cryptobellum — COMMENTED** [review]
Your work with the word complexity was really impressive and will benefit the project overall.

**Comment by BritVandy** [comment]
@singparvi I've tried it a few different ways, but it keeps telling me file not found. I'm not sure how to make it work.

**Comment by singparvi** [comment]
@BritVandy I did update the code. Kindly see the latest commit and let me know if this is still confusing. I tested it locally and the code runs without issues and the CSV is read.[Latest Commit ](https://github.com/Lambda-School-Labs/scribble-stadium-ds/commit/3af1a23e14473de8d504d4c64f6ab3769928ec8a)

**Review by Fellylove — APPROVED** [review]
Great work!

You have done a good job of commenting your code. Makes it easy to follow along!

File: app/utils/wordcloud/wordcloud_functions.py [hunk 1]
```diff
@@ -1,47 +1,42 @@
...
-    word = word.lower()
-    syllable_count = 0
-    vowels = 'aeiouy'
-    if len(word) == 0:
-        return 0
-    if word[0] in vowels:
-        syllable_count += 1
-    for index in range(1, len(word)):
-        if word[index] in vowels and word[index - 1] not in vowels:
+import re
```
Comment (peterger8y) [inline app/utils/wordcloud/wordcloud_functions.py:3]: Interesting use of regular expressions

**Review by peterger8y — COMMENTED** [review]
Good update to the word complexity

**Review by afroman32 — APPROVED** [review]
Good commenting to explain your code.
I like how you broke the words down into education level as well as complexity.

**Review by benjcrich — APPROVED** [review]
Looks awesome Brittany! Fantastic work as always!

**Review by peterger8y — APPROVED** [review]

**Pull request merged**